bot/cogs/bot.py

```diff
@@ -60,6 +65,36 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.channel.id in self.code_block_channels:
+            if self.code_block_channels[msg.channel.id]-time.time() > 300:
+                if msg.content.count("\n") >= 3:
+                    try:
+                        tree = ast.parse(msg.content)
+
+                        # Attempts to parse the message into an AST node.
+                        # Invalid Python code will raise a SyntaxError.
+                        if not all(isinstance(node, ast.Expr) for node in tree.body):
+
+                            # Multiple lines of single words could be interpreted as expressions.
+                            # This check is to avoid all nodes being parsed as expressions.
+                            # (e.g. words over multiple lines)
+                            howto = ("Please use syntax highlighted blocks, as it makes it more legible for other users.\n"
```

Comment:
This string should be split over two lines. It exceeds 120 characters. Also made a minor edit to the sentence :D
```
"Please use syntax highlighted blocks, as it makes"
"your code more legible for other users.\n"
```